src/allomancy/java/leaf/cosmere/allomancy/common/manifestation/AllomancyBrass.java

```diff
@@ -27,14 +30,67 @@ public AllomancyBrass(Metals.MetalType metalType)
 	protected void applyEffectTick(ISpiritweb data)
 	{
 		int mode = getMode(data);
-
 		String uuid = data.getLiving().getStringUUID();
-		if (mode > 0 && !playerThreadMap.containsKey(uuid))
+
+		// data thread management
 		{
-			playerThreadMap.put(uuid, new BrassThread(data));
+			if (mode > 0 && !playerThreadMap.containsKey(uuid))
```

Comment:
Same as bendalloy comments, we should probably override tick, and stick this stuff in there. After that, we can have it call the parent class tick